client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -60,33 +60,33 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 				<div className="step-store-address__info-section" />
 				<div className="step-store-address__instructions-container">
 					<TextControl
-						label={ __( 'Address line 1', 'woocommerce-admin' ) }
+						label={ __( 'Address line 1' ) }
 						value={ get( WOOCOMMERCE_STORE_ADDRESS_1 ) }
 						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_1, value ) }
 					/>
 
 					<TextControl
-						label={ __( 'Address line 2', 'woocommerce-admin' ) }
+						label={ __( 'Address line 2' ) }
 						value={ get( WOOCOMMERCE_STORE_ADDRESS_2 ) }
 						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_2, value ) }
 					/>
 
-					<FormCountrySelect
-						name="country_code"
+					<ComboboxControl
+						label={ __( 'Country / Region' ) }
 						value={ get( WOOCOMMERCE_DEFAULT_COUNTRY ) }
```

Comment:
I'm not sure why exactly, but TS believes `get( WOOCOMMERCE_DEFAULT_COUNTRY )` can return a `number`, which causes a type error if `value` must be a string. Do you believe this should be a string? If so, can you cast it to a String here?

Reply:
Is it stemming from [this](https://github.com/Automattic/wp-calypso/pull/59505/files#diff-24648f1a5849a32e3be548a0a3991dc3cafc59a3574d553d9dd75de8225aed29R40)? `as string` should be fine here right? I don't think there is any cases where a non string can enter this setting.

Reply:
@retrofox is `blog_public` required in this hook? Can't see where/how it's used (still catching up after the break) - might be faster to just remove the def.

Reply:
@sirbrillig What's the best way to catch errors like these going forward? I wasn't seeing a type error there. 🤔 

I cast it to a string here: https://github.com/Automattic/wp-calypso/pull/59876/commits/ab4b8ac7918a2eb27eb5d2a15405b87bd56c1749

> Can't see where/how it's used (still catching up after the break) - might be faster to just remove the def.
@lsl I'm not seeing this either but it seems like `get` should be able to return different types, so I'm hesitant to limit it to strings.

Reply:
Good question! Sorry I didn't explain that earlier. You'd see the type errors if the types for `Combobox` were available, but we're going to have to wait for the changes I merged there to be released and imported into calypso (and then rebased into this branch). Here's an example of when this was done recently: https://github.com/Automattic/wp-calypso/pull/59779

The reason I see them is that I manually added the types to my `node_modules` folder so I could check that they worked correctly, and that's why I was mentioning the errors to you here. This sort of thing is really weird and probably won't happen often.

Reply:
> but we're going to have to wait for the changes I merged there to be released and imported into calypso (and then rebased into this branch)

Since it's not clear to me how long this process takes, you may want to ping the calypso team and ask them to mark these unit tests as passing (assuming the missing `ComboboxControl` is still the only new error). I'd do it but I don't have that power. 😅

Reply:
> The reason I see them is that I manually added the types to my node_modules folder so I could check that they worked correctly, and that's why I was mentioning the errors to you here. 

Ahhh! Got it, thanks!